src/base/BasePaymaster.sol

```diff
@@ -40,7 +41,7 @@ abstract contract BasePaymaster is Ownable {
      * This method can also carry eth value to add to the current stake.
      * @param unstakeDelaySec - The unstake delay for this paymaster. Can only be increased.
      */
-    function addStake(uint32 unstakeDelaySec) external payable onlyOwner {
+    function addStake(uint32 unstakeDelaySec) external payable onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
I would make this admin/manager because would be difficult to collect all signatures just to add stake